lua/nvim-tundra/stylesheet/arctic.lua

```diff
@@ -64,6 +64,12 @@ stylesheet.diff = {
   text = cp.indigo._800,
 }
 
+stylesheet.filetree = {
+  bg = stylesheet.bg.floating,
+  fg = stylesheet.fg.normal,
+  border = { bg = stylesheet.bg.normal, fg = stylesheet.bg.normal },
+}
+
```

Comment:
I think this should be removed from the stylesheet. I want to keep this stylesheet as simple as possible. Check out the suggestion in `nvim-tree.lua` for my proposal!

```suggestion
```